packages/xs-extra/xapi.master/opam

```diff
@@ -51,6 +51,7 @@ depends: [
   "xapi-cli-protocol"
   "xapi-consts"
   "xapi-datamodel"
+  "xapi-expiry-alerts"
```

Comment:
@psafont Thank you for helping to add these missing part!
May I have a question: is this opam used for building the xapi binary: https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/dune#L162 ?
And may I know when will the xapi binary be requested through the opam repo?

Reply:
yes, the main xapi binary is part of the xapi opam package. The certificate alert binary is also part of the xapi opam package:
https://github.com/xapi-project/xen-api/blob/master/ocaml/alerts/certificate/dune#L18

Reply:
the opam repository is only used to test packages in github's CI and test install packages in the developers' machines. To change the binaries installed in Xenserver `xapi.spec` must be modified to include the files

Reply:
I see, thank you!